### PR TITLE
Enable debian install path for testpmd

### PR DESCRIPTION
**Pull request by mcgov**
- fixup module loading for debian to account for unneeded/missing drivers
- Fix ubuntu install_packages paths to use backports
- add debian-specific path for dependency installation

### CODE REVIEW

**Review by squirrelsc — DISMISSED** [review]

**Comment by mcgov** [comment]
Had to resolve a merge conflict, need a second approval after merge resolution

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**
